backend/Dockerfile

```diff
@@ -2,15 +2,31 @@ FROM mavromat/alpine-python:3.6
 
 LABEL maintainer="Dimitrios Mavrommatis <jim.mavrommatis@gmail.com>"
 
-RUN apk update && apk add --no-cache supervisor tcl postgresql-client postgresql-dev redis curl
+RUN apk update && apk add --no-cache supervisor tcl postgresql-client postgresql-dev redis curl git libffi-dev cmake libssh
 
 WORKDIR /root
 
 COPY . ./
 
+RUN git clone https://github.com/rtrlib/rtrlib.git
+WORKDIR /root/rtrlib
+RUN cmake -D CMAKE_BUILD_TYPE=Release .
+RUN make && make install
+WORKDIR /root
+
 RUN pip --no-cache-dir install -r requirements.txt \
-    && pip --no-cache-dir install git+https://github.com/supervisor/supervisor@a0ee8f1026c929ae4d9fc84741924414e8008f49 \
-    && make clean && make -j 4 \
+    && pip --no-cache-dir install git+https://github.com/supervisor/supervisor@a0ee8f1026c929ae4d9fc84741924414e8008f49
+
+RUN ln -s /usr/local/lib64/librtr.so /usr/local/lib/librtr.so \
+    && ln -s /usr/local/lib64/librtr.so.0 /usr/local/lib/librtr.so.0 \
+    && ln -s /usr/local/lib64/librtr.so.0.7.0 /usr/local/lib/librtr.so.0.7.0
+
+RUN git clone https://github.com/rtrlib/python-binding.git
```

Comment:
same as above

Reply:
how can I lock this from source to a specific version?

Reply:
done. Resolving